src/web/vaev-style/values.cpp

```diff
@@ -113,6 +113,52 @@ Res<bool> ValueParser<bool>::parse(Cursor<Css::Sst>& c) {
 }
 
 // MARK: Border-Style
+static Res<CalcValue<Length>> _parseLineWidth(Cursor<Css::Sst>& c) {
+    if (c.peek() == Css::Token::ident("thin")) {
```

Comment:
code duplication here from BorderProps, will fix this once done with BorderCollapse